recipes/libproperties/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+
+
+class LibpropertiesConan(ConanFile):
+    name = "libproperties"
+    version = "1.1.2"
+    license = "Apache-2.0 License"
+    author = "libbylg@126.com"
+    url = "https://github.com/tinyhubs/libproperties"
+    description = "libproperties is a library to parse the Java .properties files. It was writen in pure C. And fullly compatible with the Java .properties file format."
+    topics = (".properties", "Java", "pure C")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+
+    def source(self):
+        self.run("git clone https://github.com/tinyhubs/libproperties.git")
+        # This small hack might be useful to guarantee proper /MT /MD linkage
+        # in MSVC if the packaged project doesn't have variables to set it
+        # properly
+        tools.replace_in_file("libproperties/CMakeLists.txt", "PROJECT(libproperties)",
+                              '''PROJECT(libproperties)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()''')
+
+    def build(self):
+        cmake = CMake(self)
```

Comment:
I can't find a cmake script at https://github.com/tinyhubs/libproperties
Only a premake for its tests.
I guess you will need to create a small cmake script yourself:

```cmake
cmake_minimum_required(VERSION 2.8)
project(libproperties C)
include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
conan_basic_setup()
add_library(libproperties buf.c properties.c)
install(TARGETS properties RUNTIME DESTINATION bin LIBRARY DESTINATION lib ARCHIVE DESTINATION lib)
install(FILES properties.h DESTINATION include)
```
I used target `properties` instead of `libproperties` as this will create a file `libproperties.a` using gcc.

Reply:
I'm sorry, I make a mistake.  I'm not pull the dev branch.

Reply:
Ah, yes. There it is!
Is it ok if I open a pr with some suggestions to make it more cross platform compatible?

Reply:
https://github.com/tinyhubs/libproperties/pull/2